#! /bin/sh -e
##
## DP: Modify the lynx.cfg to Debian system

--- ./lynx.cfg.orig	2010-06-26 12:06:05.000000000 +0900
+++ ./lynx.cfg	2010-06-26 14:13:18.000000000 +0900
@@ -90,7 +90,7 @@
 #
 # Normally we expect you will connect to a remote site, e.g., the Lynx starting
 # site:
-STARTFILE:http://lynx.isc.org/
+#STARTFILE:http://lynx.isc.org/
 #
 # As an alternative, you may want to use a local URL.  A good choice for this is
 # the user's home directory:
@@ -114,9 +114,10 @@
 # This definition will be overridden if the "LYNX_HELPFILE" environment
 # variable has been set.
 #
-HELPFILE:http://lynx.isc.org/release/breakout/lynx_help/lynx_help_main.html
+#HELPFILE:http://lynx.isc.org/release/breakout/lynx_help/lynx_help_main.html
 .ex
-#HELPFILE:file://localhost/PATH_TO/lynx_help/lynx_help_main.html
+##HELPFILE:file://localhost/PATH_TO/lynx_help/lynx_help_main.html
+HELPFILE:file://localhost/usr/share/doc/lynx-cur/lynx_help/lynx_help_main.html
 
 .h2 DEFAULT_INDEX_FILE
 # DEFAULT_INDEX_FILE is the default file retrieved when the
@@ -436,7 +437,7 @@
 # Find RFC 1345 at
 .url http://tools.ietf.org/html/rfc1345
 #
-#CHARACTER_SET:iso-8859-1
+CHARACTER_SET:iso-8859-1
 
 .h2 LOCALE_CHARSET
 # LOCALE_CHARSET overrides CHARACTER_SET if true, using the current locale to
@@ -450,6 +451,7 @@
 # happens to give useful values, but other implementations are not guaranteed
 # to do this.
 #LOCALE_CHARSET:FALSE
+LOCALE_CHARSET:TRUE
 
 .h2 HTML5_CHARSETS
 # HTML5_CHARSETS is an alternative to ASSUME_CHARSET and ASSUME_LOCAL_CHARSET.
@@ -547,7 +549,7 @@
 # compatibility problems with other browsers, see also PREPEND_BASE_TO_SOURCE.
 # Note that the prepending is not done for -source dumps.
 #
-#PREPEND_CHARSET_TO_SOURCE:TRUE
+PREPEND_CHARSET_TO_SOURCE:FALSE
 
 .h2 NCR_IN_BOOKMARKS
 # NCR_IN_BOOKMARKS:TRUE allows you to save 8-bit characters in bookmark titles
@@ -599,7 +601,7 @@
 # the 'o'ptions menu and save that preference in their RC file.
 # This may be a comma-separated list of languages in decreasing preference.
 #
-#PREFERRED_LANGUAGE:en
+PREFERRED_LANGUAGE:en
 
 .h2 PREFERRED_CHARSET
 # PREFERRED_CHARSET specifies the character set in MIME notation (e.g.,
@@ -791,7 +793,7 @@
 # 'o'ptions menu and saved in the RC file, and always can be toggled
 # via the -show_cursor command line switch.
 #
-#SHOW_CURSOR:FALSE
+SHOW_CURSOR:TRUE
 
 .h2 UNDERLINE_LINKS
 # UNDERLINE_LINKS controls whether links are underlined by default, or shown
@@ -979,8 +981,8 @@
 # executed.  This is much less dangerous than enabling all execution
 # links, but can still be dangerous.
 #
-#LOCAL_EXECUTION_LINKS_ALWAYS_ON:FALSE
-#LOCAL_EXECUTION_LINKS_ON_BUT_NOT_REMOTE:FALSE
+LOCAL_EXECUTION_LINKS_ALWAYS_ON:FALSE
+LOCAL_EXECUTION_LINKS_ON_BUT_NOT_REMOTE:FALSE
 
 .h2 TRUSTED_EXEC
 # If LOCAL_EXECUTION_LINK_ON_BUT_NOT_REMOTE is TRUE, and no TRUSTED_EXEC
@@ -1027,7 +1029,7 @@
 # as well (i.e., they will only be able to access lynxexec or lynxprog
 # URLs which meet any ALWAYS_TRUSTED_EXEC rules).
 #
-#TRUSTED_EXEC:none
+TRUSTED_EXEC:none
 
 .h2 ALWAYS_TRUSTED_EXEC
 # If EXEC_LINKS was defined, any lynxexec or lynxprog URL can be made
@@ -1048,7 +1050,7 @@
 #
 # The default ALWAYS_TRUSTED_EXEC rule is "none".
 #
-#ALWAYS_TRUSTED_EXEC:none
+ALWAYS_TRUSTED_EXEC:none
 
 .h2 TRUSTED_LYNXCGI
 # Unix:
@@ -1067,7 +1069,7 @@
 #
 # The default TRUSTED_LYNXCGI rule is "none".
 #
-#TRUSTED_LYNXCGI:none
+TRUSTED_LYNXCGI:none
 
 .h2 LYNXCGI_ENVIRONMENT
 # Unix:
@@ -1387,7 +1389,7 @@
 # The default is FALSE, so that the feature needs to be enabled here
 # explicitly if you want it.
 #
-#PERSISTENT_COOKIES:FALSE
+PERSISTENT_COOKIES:FALSE
 
 .h2 COOKIE_FILE
 # COOKIE_FILE is the default file from which persistent cookies are read
@@ -1834,7 +1836,7 @@
 # and not forced TRUE, the user can regulate it via the 'o'ptions menu (and
 # may save the preference in the RC file).
 #
-#NO_DOT_FILES:TRUE
+NO_DOT_FILES:FALSE
 
 .h1 Internal Behavior
 
@@ -2099,13 +2101,13 @@
 #
 # Unix:
 # ====
-#GLOBAL_EXTENSION_MAP:/usr/local/lib/mosaic/mime.types
+GLOBAL_EXTENSION_MAP:/etc/mime.types
 # VMS:
 # ===
 #GLOBAL_EXTENSION_MAP:Lynx_Dir:mime.types
 #
 #	Unix (sought in user's home directory):
-#PERSONAL_EXTENSION_MAP:.mime.types
+PERSONAL_EXTENSION_MAP:.mime.types
 #	VMS (sought in user's sys$login directory):
 #PERSONAL_EXTENSION_MAP:mime.types
 
@@ -2240,6 +2242,8 @@
 #SUFFIX:.dms:application/x-dms
 #SUFFIX:.html:text/html
 #SUFFIX:.txt:text/plain
+SUFFIX:.tgz:application/octet-stream
+SUFFIX:.deb:application/octet-stream
 
 .h2 XLOADIMAGE_COMMAND
 # VMS:
@@ -2343,13 +2347,13 @@
 #
 # Unix:
 # ====
-#GLOBAL_MAILCAP:/usr/local/lib/mosaic/mailcap
+GLOBAL_MAILCAP:/etc/mailcap
 # VMS:
 # ===
 #GLOBAL_MAILCAP:Lynx_Dir:mailcap
 #
 # 	Sought in user's home (Unix) or sys$login (VMS) directory.
-#PERSONAL_MAILCAP:.mailcap
+PERSONAL_MAILCAP:.mailcap
 
 .h2 PREFERRED_MEDIA_TYPES
 # When doing a GET, lynx lists the MIME types which it knows how to present
@@ -3554,7 +3558,7 @@
 # should accept, in case the $SSL_CERT_FILE environment variable is not set,
 # e.g.,
 #
-#SSL_CERT_FILE:/etc/ssl/certs/ca-certificates.crt
+SSL_CERT_FILE:/etc/ssl/certs/ca-certificates.crt
 #SSL_CERT_FILE:NULL
 
 .h1 Appearance
@@ -3578,7 +3582,7 @@
 
 .h2 SYSLOG_REQUESTED_URLS
 # Log the requested URLs using the syslog interface.
-#SYSLOG_REQUESTED_URLS:TRUE
+SYSLOG_REQUESTED_URLS:FALSE
 
 .h2 SYSLOG_TEXT
 # Add the given text to calls made to syslog, to distinguish Lynx from other
@@ -3744,3 +3748,19 @@
 .h2 WITH_BACKSPACES
 # Emit backspaces in output if -dump'ing or -crawl'ing (like 'man' does).
 #WITH_BACKSPACES:FALSE
+
+# external
+EXTERNAL:ftp:w3m %s:TRUE
+EXTERNAL:file:w3m %s:TRUE
+EXTERNAL:http:w3m %s:TRUE
+EXTERNAL:http:wget %s:TRUE
+EXTERNAL:http:wget -r %s:TRUE
+EXTERNAL:ftp:x-www-browser %s:TRUE
+EXTERNAL:file:x-www-browser %s:TRUE
+EXTERNAL:http:x-www-browser %s:TRUE
+# include
+INCLUDE:/etc/lynx-cur/local.cfg
+INCLUDE:~/.lynx/colors:COLOR
+INCLUDE:~/.lynx/keymap:KEYMAP
+INCLUDE:~/.lynx/viewers:VIEWER
+INCLUDE:~/.lynx/external:EXTERNAL
